server/container_create_linux.go

```diff
@@ -38,7 +35,6 @@ import (
 )
 
 // Copied from k8s.io/kubernetes/pkg/kubelet/kuberuntime/labels.go
```

Comment:
The const is moved out of here, and this ^^^ comment (that was about the const) feels lonely. Please help it to reunite with the const it describes.

Reply:
fixed !